src/nucleic_acid/nucleic_acid.jl

```diff
@@ -55,28 +45,28 @@ end
 
 
 @inline function P_generic(mod::NASM, t::Float64)
-  if t < 0.0
-    error("t must be positive")
-  end
-  return expm(Q(mod) * t)
+    if t < 0.0
+        error("t must be positive")
```

Comment:
I think this is fine for now, but in the future we will want to throw an actual exception type, as it seems throwing a "DomainError" kind of exception is relevant in this cases where a value must be positive.